Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.36740&attempt_id=d8f6f86f-2e6d-43f7-8660-a5301d350eb2)

<details><summary>Partial log (click to expand)</summary><p>

```
                 from ./db/filename.h:14,
                 from db/builder.cc:7:
./port/atomic_pointer.h:212:2: error: #error Please implement AtomicPointer for this platform.
 #error Please implement AtomicPointer for this platform.
  ^~~~~
make[1]: *** [Makefile:202: db/builder.o] Error 1
make[1]: Leaving directory '/build/memorycoin-0.8.5/src/leveldb'
make: *** [Makefile:382: /build/memorycoin-0.8.5/src/leveldb/libleveldb.a] Error 2
builder for '/nix/store/rybn21c6bmxjglrl3z20xl04l7hc6mp6-memorycoin-0.8.5.drv' failed with exit code 2
[31;1merror:[0m build of '/nix/store/rybn21c6bmxjglrl3z20xl04l7hc6mp6-memorycoin-0.8.5.drv' failed
```
</p></details>

--- a/pkgs/applications/altcoins/default.nix
+++ b/pkgs/applications/altcoins/default.nix
@@ -46,8 +46,8 @@ rec {
   litecoin  = callPackage ./litecoin.nix { withGui = true; };
   litecoind = callPackage ./litecoin.nix { withGui = false; };
 
-  memorycoin  = callPackage ./memorycoin.nix { withGui = true; };
-  memorycoind = callPackage ./memorycoin.nix { withGui = false; };
+  memorycoin  = callPackage ./memorycoin.nix { boost = boost165; withGui = true; };
+  memorycoind = callPackage ./memorycoin.nix { boost = boost165; withGui = false; };
 
   namecoin  = callPackage ./namecoin.nix  { withGui = true; };
   namecoind = callPackage ./namecoin.nix { withGui = false; };

--- a/pkgs/applications/altcoins/memorycoin.nix
+++ b/pkgs/applications/altcoins/memorycoin.nix
@@ -51,6 +51,6 @@ stdenv.mkDerivation rec{
     homepage = http://www.bitcoin.org/;
     maintainers = with maintainers; [ AndersonTorres ];
     license = licenses.mit;
-    platforms = platforms.unix;
+    platforms = subtractLists [ "aarch64-linux" ] platforms.unix;
   };
 }

